### ISSUE TITLE
Allow pipeline coverage data to provide CI coverage support on native browser

### ISSUE THREAD

**Issue by bitinn** [defunctzombie/zuul#217]
When we need to report coverage data to a third-party service, such as [coveralls](https://coveralls.io/), we would like to rely on `zuul` command to pipe us the coverage result from our sauce labs tests (allow us to specify which browser's test result to use, for example.)

Currently we have to install [mochify](https://github.com/mantoni/mochify.js) in addition to `zuul`, in order to use `phantomjs` to run the same tests to produce coverage result.

And for one of my project, [vdom-parser](https://github.com/bitinn/vdom-parser), this becomes a problem because we rely on DOMParser API in browsers, which `phantomjs` only has partial support.

So we ended up needing polyfill for DOMParser API, and because it's not spec compliant, we are unable to achieve 100% test coverage, even though on supported browsers we do have 100% coverage.

TL;DR: follow up on #174, please help us to get coverage data from real browsers and allow us to pipe them to coverage reporting services.

**Comment by cesarandreu** [comment]
Since you wanna get code coverage from the tests running on the actual browser (and not phantomjs), can't you hit the `http://localhost:58700/__zuul/coverage/download` endpoint?

**Comment by bitinn** [comment]
I can try, to pull stats archive using curl and then unzip it and then pipe it to coveralls. Not sure if it would work on sauce labs though.

**Comment by defunctzombie** [comment]
I would review a PR or other improvements to make this work for you and other users. I don't recall the details of the coverage additions but if something is missing we can add it.
